netgrasp/netgrasp.py

```diff
@@ -10,17 +10,19 @@
 import logging
 import logging.handlers
 import pwd
-import sys
 import os
 import datetime
 import time
-import traceback
 
 netgrasp_instance = None
 
 BROADCAST = 'ff:ff:ff:ff:ff:ff'
 
-ALERT_TYPES = ['requested_ip', 'first_requested_ip', 'first_requested_ip_recently', 'seen_device', 'first_seen_device', 'first_seen_device_recently', 'seen_mac', 'first_seen_mac', 'seen_ip', 'first_seen_ip', 'seen_host', 'first_seen_host', 'seen_vendor', 'first_seen_vendor', 'device_stale', 'request_stale', 'changed_ip', 'duplicate_ip', 'duplicate_mac', 'network_scan', 'ip_not_on_network', 'src_mac_broadcast', 'requested_self']
+ALERT_TYPES = ['requested_ip', 'first_requested_ip', 'first_requested_ip_recently', 'seen_device', 'first_seen_device',
+               'first_seen_device_recently', 'seen_mac', 'first_seen_mac', 'seen_ip', 'first_seen_ip', 'seen_host',
+               'first_seen_host', 'seen_vendor', 'first_seen_vendor', 'device_stale', 'request_stale', 'changed_ip',
+               'duplicate_ip', 'duplicate_mac', 'network_scan', 'ip_not_on_network', 'src_mac_broadcast',
+               'requested_self']
```

Comment:
> unused imports/variables

I love removing cruft.

>  variable names that clash with python built-in names (e.g. "range")

Usually I don't do this as vim highlights Python keywords.

> variable names that shadow other names (e.g. function or imported names)

This has caused bugs in netgrasp before (something like "using variable before it was declared") hence why I merged all those changes.

> I'll do that for the things you're not concerned with.

Long lines is the only item I'm not really a fan of, but I'm willing to follow the pep8 coding standards. I prefer to embrace existing coding standards versus creating our own.